src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSSemanticConventions.cs

```diff
@@ -12,6 +12,9 @@ internal static class AWSSemanticConventions
 
     public const string AttributeAWSDynamoTableName = "aws.table_name";
     public const string AttributeAWSSQSQueueUrl = "aws.queue_url";
+    public const string AttributeAWSSQSQueueName = "aws.sqs.queue_name";
+    public const string AttributeAWSS3BucketName = "aws.s3.bucket";
+    public const string AttributeAWSKinesisStreamName = "aws.kinesis.stream_name";
```

Comment:
If there is no more generic concept, please define it fist on semantic convention level.
Without this, it can be easily out of sync with other languages.

Reply:
Same as above, these attribute names:
* Aligns with the AWS SDK instrumentation in other languages
* Follows the naming convention of aws.$SERVICE.$RESOURCE_NAME
* Aligns with OTel semantic conventions, if already available:
  -  For example,[ S3 bucket semantics](https://opentelemetry.io/docs/specs/semconv/object-stores/s3/)
* AWS SDK team is currently working on defining all AWS resource attributes at the semantic convention level